docs/src/pages/languages/python.mdx

```diff
@@ -7,17 +7,27 @@ edit: false
 
 # Python SDK
 
-Our Python SDK is designed to easily integrate into existing the database drivers workflow. All you have to do is use the `connect` method it provides. For example, if you are using Psycopg2, all you have to do to adopt Approzium into your client code base is:
+Our Python SDK is designed to easily integrate into existing the database drivers workflow. All you have to do is use
+the `connect` method it provides. For example, if you are using Psycopg2, all you have to do to adopt Approzium into
+your client code base is:
 
 ```diff
 -from psycopg2 import connect
 +from approzium.psycopg2 import connect
-+from approzium import AuthClient
++from approzium import AuthClient, TLSConfig
 
 -conn = connect(password=verySecurePassword", ...)
-+conn = connect(authenticator=AuthClient("authenticator:6001") ...)
++conn = connect(authenticator=AuthClient(
+    "authenticator:6001", TLSConfig(
+        trusted_certs="path/to/approzium.cert",
+        client_cert="path/to/client.pem",
+        client_key="path/to/client.key"
+    ), ...
```

Comment:
I am of the opinion that the TLS should be stripped out, since this is just pseudocode that is trying to show the general idea of how the connect method works.

Reply:
Sure, good point. How about up at the top I change it to `disable_tls`, and below where I get into the TLSConfig, I move this example? That way we show the easy way first, and then we have an example below of how we want them to do it.

Reply:
This is done in https://github.com/cyralinc/approzium/pull/111/commits/b2294a33113af0fb2a2b8733793820803a4fc3e9 and https://github.com/cyralinc/approzium/pull/111/commits/7e0e744fdb330def1987779775e99d007adae13c.